Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -215,7 +215,7 @@ public Vector3[] TryGetBoundaryRectanglePoints()
             var positions = new Vector3[points2d.Length];
             for (int i = 0; i < points2d.Length; ++i)
             {
-                positions[i] = new Vector3(points2d[i].x, boundaryFloor, points2d[i].y);
+                positions[i] = CameraCache.Main.transform.parent.TransformPoint(new Vector3(points2d[i].x, boundaryFloor, points2d[i].y));
```

Comment:
This change isn't relevant to custom cursors. Please, either make a new PR and revert this change, or add it to the description.

Reply:
This change was made by @keveleigh.

Reply:
Oh, I see, you merged from a different branch into this one?

Reply:
Yes. I took june18_dev branch as base.

Reply:
Sure. Will do that.

Reply:
Thanks.

Reply:
The changes made in #2518 are already there in my branch.